src/hotspot/share/opto/compile.cpp

```diff
@@ -5079,6 +5092,9 @@ void Compile::print_method(CompilerPhaseType cpt, int level, Node* n) {
   ResourceMark rm;
   stringStream ss;
   ss.print_raw(CompilerPhaseTypeHelper::to_description(cpt));
+  if (iter > 0) {
+    ss.print(" %d", iter);
+  }
```

Comment:
I agree with just having a `print_method`. I initially hesitated to modify `print_method` in the way you suggest, as it would then add iteration numbering to _all_ phases with no exceptions. But, it seems this is a feature we want then?

Reply:
I think it could be useful for any repeated phase, if others also agree with that. And we would still keep the same name for phases that are only printed once and only add a number for the repeated dumps.

Reply:
I've changed it for now. If anyone does not agree, please let us know.